### PR TITLE
Host the Awin javascript file on our own server

### PR DESCRIPTION
**Pull request by rupertbates**
To avoid hosting Javascript files which we don't control on our checkout flows we have decided to host Awin's js on our own servers. This means that should their script file become compromised we won't be affected.

### CODE REVIEW

File: app/views/main.scala.html [hunk 1]
```diff
@@ -47,6 +48,6 @@
         @fragments.global.footer(product, edition, contactUsCountry)
         @fragments.footerJavaScript()
         <!--AWIN affiliate tracking tag - see here: https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW -->
-         <script src="https://www.dwin1.com/11152.js" type="text/javascript" defer="defer"></script>
+         <script src="@hashedPathFor("javascripts/vendor/11152.js")" type="text/javascript" defer="defer"></script>
```
Comment (akash1810) [inline app/views/main.scala.html]: Could we use a more obvious filename for this self hosted asset to make it obvious what vendor its for?

File: assets/javascripts/vendor/awin-11152.js [hunk 2]
```diff
@@ -0,0 +1 @@
...8|0x4d2c6dfc|0x53380d13|0x650a7354|0x27b70a85|0x14292967|0xc6e00bf3|0xd5a79147|0x06ca6351|0x90befffa|0xa4506ceb|2000|apply|INPUT|QX4QkKEU|setTimeout|0x1f|u00ef|0x0f|u00c0|u00df|about|blank|Infinity|max|awin_tntc|yes|Object|defined|TypeError|or|not|u00e0||||||decode||0xa54ff53a|0x510e527f|||||||||||0x9b05688c|||0x1f83d9ab||0x3c6ef372|z_|0xbef9a3f7|0xc67178f2|0x6a09e667|0x5be0cd19|ceil|u0800|uffff|0xe0|0x3F|0xc0|u07ff|pow|floor|0xf|0xc19bf174|0xbb67ae85|0x9bdc06a7|OrderID|OrderTotal|SiteID|AdvID|get|Error|oError|ia|255|basket|navigator|appName|Microsoft|Internet|category|quantity|product_line|||getBasketData||||||||price|loaded||||complete|365||||1000|||expires|||toGMTString|getTime|setTime|Z_|parseanchorAtp|Date|path|gclid_|img|_aw_script_|onreadystatechange|pixelAppend|frameAppend|getAffiliateId|head|html|Explorer|1062|0x428a2f98|err|getAttribute|confirm|0x71374491|0xe9b5dba5|0xab1c5ed5|0x923f82a4|0x59f111f1|0x3956c25b|content|XPathResult|CCampID|11152|ImpID|gv|3055|CampID||ANY_TYPE|||iterateNext|||||node||||||selectNodes|0xd807aa98|||0xb5c0fbcf|||fetchZxParam||||instanceof|innerHTML|cks|0x72be5d74|META|zx|href|0x243185be|0x12835b01|0x80deb1fe|0x550c7dc3|pv|parts'.split('|'),0,{}))
```
Comment (akash1810) [inline assets/javascripts/vendor/awin-11152.js]: `eval` 😱 I really don't trust this script!

Comment (rupertbates) [reply assets/javascripts/vendor/awin-11152.js]: It/Awin has been audited by the Infosec team

File: app/views/main.scala.html [hunk 3]
```diff
@@ -1,7 +1,8 @@
-@import com.gu.i18n.Country
+@@import com.gu.i18n.Country
```
[the remark this answers is not in the document]
Comment (rupertbates) [reply app/views/main.scala.html]: 😕 Not sure where that came from!

**Comment by LATaylor-guardian** [comment]
👍

**Pull request merged**